redi-connect-front/src/pages/front/signup-existing/SignUpExisting.tsx

```diff
@@ -1,20 +1,20 @@
-import React, { useEffect, useState } from "react";
-import { LoggedOutLayout } from "../../../layouts/LoggedOutLayout";
-import { RouteComponentProps } from "react-router";
-import { useLoading } from "../../../hooks/WithLoading";
-import { saveAccessToken } from "../../../services/auth/auth";
-import { fetchSaveRedProfile } from "../../../services/api/api";
-import { history } from "../../../services/history/history";
+import React, { useEffect, useState } from 'react';
```

Comment:
Let's completely remove the folder `signup-existing` and the routes related to it. Or, if the files are useful files to you, at least tag them to be "refactored away" in a later ticket.

It's a very old form that was only used right after we initially launched ReDI Connect. The users were imported into the DB from some typeform data, and sent emails to activate their users.

Reply:
Task created

Reply:
Perfect, thanks. Resolving convo.